.github/workflows/preview.yml

```diff
@@ -80,8 +90,98 @@ jobs:
+
+  build_windows:
+    name: Build Windows standalone binary
+    if: inputs.CompileWindows
+    needs: publish_preview
+    runs-on: windows-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+        # Nuitka needs the packaged form and not the editable install Poetry provides
+        # Ref: https://github.com/Nuitka/Nuitka/issues/2965
+      - name: Download build artifacts
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+
+      - name: Set up Python
+        uses: actions/setup-python@f677139bbe7f9c59b41e40162b753c062f5d49a3 # v5.2.0
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Install poetry
+        run: pipx install --python ${{ env.PYTHON_VERSION }} poetry==${{ env.POETRY_VERSION }}
+
+      - name: Configure poetry
+        run: poetry config virtualenvs.in-project true
+
+      - name: Install the project with poetry
+        run: |
+          poetry check --lock
+          poetry lock --no-update --no-cache
+          poetry install --verbose --no-root --sync --with compile
+          poetry run python -m pip install --find-links dist --no-index phylum
+
+      - name: Compile binary with Nuitka
+        env:
+          PREVIEW_VER: ${{ needs.publish_preview.outputs.next_ver }}
+        run: |
+          poetry run python -m nuitka `
+            --onefile `
+            --output-dir=build `
+            --output-filename="phylum-ci.exe" `
+            --include-package=phylum `
+            --include-package-data=phylum `
+            --include-distribution-metadata=phylum `
+            --onefile-tempdir-spec="{CACHE_DIR}/{PRODUCT}/{VERSION}" `
+            --product-name=phylum-ci `
+            --product-version=${env:PREVIEW_VER} `
+            --file-version=${env:GITHUB_RUN_NUMBER} `
+            --company-name="Phylum, Inc." `
+            --copyright="Copyright (C) 2024 Phylum, Inc." `
+            --file-description="Analyze dependencies in CI with Phylum" `
+            --windows-icon-from-ico="docs/img/favicon.ico" `
+            --warn-implicit-exceptions `
+            --warn-unusual-code `
+            --assume-yes-for-downloads `
+            --report=nuitka-compilation-report.xml `
+            --deployment `
+            src/phylum/ci/cli.py
+
+      - name: Confirm operation of binary
+        env:
+          PHYLUM_API_KEY: ${{ secrets.PHYLUM_TOKEN }}
+          PHYLUM_BYPASS_CI_DETECTION: true
+        run: |
+          ./build/phylum-ci.exe -h
+          ./build/phylum-ci.exe -vvaf
+
+      - name: Upload standalone binary
+        if: always()
+        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
+        with:
+          name: phylum-ci.exe
+          path: ./build/phylum-ci.exe
+          if-no-files-found: error
```

Comment:
Do you know how "standalone" this binary is? Is it a completely statically linked python interpreter?

Reply:
Yes. This binary operates completely independently. Python does not already need to be installed on the system.